### ISSUE TITLE
updated homepage design

### ISSUE THREAD

**Issue by katiewilson21** [ResetNetwork/recoding-tech#106]
see here: https://www.figma.com/file/L9szOsvQdhyumMOGEz4VnI/?node-id=0%3A1 "New Homepage"

Includes:

- updated title styling (matches existing styling for the policy tracker modules) 
- ability to display 3 featured articles 
- abbreviated policy tracker module
- newsletter button in right column

let me know if you have any questions!

**Comment by harumhelmy** [comment]
Hi @katiewilson21 @yaelrst I'm knocking out some of the other issues (most of those are trivial!) and will get to this one, which is non-trivial, early next week. Estimating they'll be up on the test site by Thursday.

**Comment by harumhelmy** [comment]
A bunch of these changes are now up on the test site, @katiewilson21 @yaelrst!! There's some minor details I still need to work out in terms of margins, sizing breakpoints, making the logo in the Subscribe section smaller, etc. Estimating I'll get to those by Friday.

**Comment by katiewilson21** [comment]
@harumhelmy You could just change the logo image in the subscribe box to the text "Recoding.Tech" if that makes it easier

**Comment by katiewilson21** [comment]
looks really good! -- what size is the font of the italics subtitles? (like "New issues, policies, governments, and companies we’re covering" etc)? looks small but I might be wrong (should be 14pt)

**Comment by jheretic** [comment]
Hi @katiewilson21, I've pushed up the requested changes to the test site if you want to take a look, especially if the fonts look right (at the specified sizes they look a bit bigger proportionally than in the screenshot, but that might just be me).

**Comment by katiewilson21** [comment]
@jheretic @harumhelmy yeah, the fonts look too big which is odd ... is the text size of the right column titles (ie "latest headlines and highlights") 14pt? If so, can we make that 12pt (in addition to the text in the newsletter subscribe box) and make the titles for the main column 16pt and see how that looks...

**Comment by katiewilson21** [comment]
Can we also decrease the spacing between title and subtitle on the main column by like 10px?

**Comment by katiewilson21** [comment]
@harumhelmy everything looks good! last thing is it might be better to switch this little logo back to just plain type with a red dot between bc it's looking wonky 
<img width="174" alt="image" src="https://user-images.githubusercontent.com/84089903/171866012-e30bc951-2b86-4db3-adf6-7a7c8547f465.png">

**Comment by katiewilson21** [comment]
(that's in the newsletter sign up box)

**Comment by yaelrst** [comment]
Edits from Ben! 

@harumhelmy @katiewilson21 

![Website__revisions_June2022](https://user-images.githubusercontent.com/100793398/172213547-eaca54f1-e4d0-443b-b0f6-f72df0e9dbbc.jpg)

**Comment by yaelrst** [comment]
Adding the different pieces of feedback (images are more detailed) as a check list: 

**Home page edits:**
- [ ] Adding more space above featured topics
- [ ] Aligning beige monthly update box with featured topics
- [ ] Reducing size of italic subtitle font for all elements 
- [ ] Checking the fonts on the right and left column (commentary and headlines) and choosing one 
- [ ] Reduce size of both View All text on the homepage 
- [ ] Make right column narrower to create more space between the right and left columns 
- [ ] Unbold title of items in tracker on homepage 
- [ ] remove type column of the tracker on the homepage 

**Topic page edits:**
- [ ] Unbold title of items in tracker 
- [ ] Left indent align columns in tracker instead of center align

**Comment by gt150** [comment]
@harumhelmy let's pause moving forward on these until Katie and I can discuss. Sorry for any confusion.
